### PR TITLE
Added brew reference

### LINKED ISSUE #1051: Add brew cask to install cli?
[Brew](https://brew.sh/) is used to installing various utilities on macos, it will be nice to use it to install the CLI.

### PR DESCRIPTION
**Pull request by omerlh**
Signed-off-by: omerlh <omerl@soluto.com>

fix #1051
based on #1056

### CODE REVIEW

**Comment by omerlh** [comment]
I did it using a cherry pick so I probably broke something - on the other hand, I don't see conflicts. Can you help?

**Comment by omerlh** [comment]
Ok, I think I tried any git voodo I know and it's finally ready :)

File: docs/get-started.md [hunk 1]
```diff
@@ -26,6 +26,13 @@ See [Set up Ark on your platform][3] for how to configure Ark for a production e
 
 1. Move the `ark` binary from the Ark directory to somewhere in your PATH.
 
+#### MacOS Installation
+
+On Mac you can use HomeBrew to install ark CLI, by running the following command:
+```
```
Comment (ncdc) [inline docs/get-started.md]:
```suggestion
```bash
```

File: docs/get-started.md [hunk 2]
```diff
@@ -26,6 +26,13 @@ See [Set up Ark on your platform][3] for how to configure Ark for a production e
 
 1. Move the `ark` binary from the Ark directory to somewhere in your PATH.
 
+#### MacOS Installation
+
+On Mac you can use HomeBrew to install ark CLI, by running the following command:
```
Comment (ncdc) [inline docs/get-started.md]:
```suggestion
On Mac, you can use [HomeBrew](https://brew.sh) to install the `ark` client:
```

**Comment by omerlh** [comment]
Done :)

**Pull request merged**